api/kv_v1.go

```diff
@@ -13,6 +13,8 @@ type KVv1 struct {
 	mountPath string
 }
 
+const Kvv1MountType = "kvv1"
```

Comment:
These can go away now right?

Reply:
Yup! Fixed this just as you commented :)